### PR TITLE
Made grunt to log each file only on verbose

### PR DESCRIPTION
**Pull request by ascripcaru**
@vladikoff please take a look at this change. I am working on a project that outputs only for this task about __3600__ lines of log in travis and is hard to follow the log. And also this PR fixes this issue https://github.com/gruntjs/grunt-contrib-compress/issues/161

### CODE REVIEW

**Comment by ascripcaru** [comment]
@vladikoff please take a look at this change. I am working on a project that outputs only for this task about __3600__ lines of log in travis and is hard to follow the log. And also this PR fixes this issue https://github.com/gruntjs/grunt-contrib-compress/issues/161

**Comment by XhmikosR** [comment]
Personally, I don't like that. No feedback is bad user experience.

I think the best solution would be to check for a number of files, say 10, and if they are more than that then show the rest in verbose mode only.

**Comment by ErikDubbelboer** [comment]
Just log one line with the number of files and in verbose one line per file.

**Comment by ascripcaru** [comment]
@ErikDubbelboer other modules didn't had that feature, until now

image

File: tasks/lib/compress.js [hunk 1]
```diff
@@ -34,16 +34,19 @@ module.exports = function(grunt) {
   // 1 to 1 gziping of files
   exports.gzip = function(files, done) {
     exports.singleFile(files, zlib.createGzip, 'gz', done);
+    grunt.log.writeln('Created ' + chalk.cyan(files.length) + ' file(s)');
```
Comment (XhmikosR) [inline tasks/lib/compress.js]: Might be worth using `util.pluralize`.

**Comment by ascripcaru** [comment]
@XhmikosR done. Also changed from `Created` to `Compressed` because you can have one archive with multiple files in it so the previous message wasn't correct

**Comment by XhmikosR** [comment]
@ascripcaru: please squash the patches.

/CC @vladikoff

**Comment by vladikoff** [comment]
Awesome, yeah squash into one and we will merge it up!

**Comment by ascripcaru** [comment]
Closing this in favor of https://github.com/gruntjs/grunt-contrib-compress/pull/165
made a mess here :smile:
